.protolint.yaml

```diff
@@ -3,6 +3,7 @@ lint:
     all_default: true
     remove:
       - FIELD_NAMES_LOWER_SNAKE_CASE
+      - ENUM_FIELD_NAMES_ZERO_VALUE_END_WITH
```

Comment:
The reason why I removed this lint is because in our case we are expecting a repeated of an enum, meaning that if user does not set anything, this field will not show up, hence does not make sense to support a `UNSPECIFIED`.